src/components/EmojiPicker/index.js

```diff
@@ -33,6 +33,15 @@ class EmojiPicker extends React.Component {
         this.emojiPopoverDimensionListener = Dimensions.addEventListener('change', this.measureEmojiPopoverAnchorPosition);
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        if (prevState.isEmojiPickerVisible === this.state.isEmojiPickerVisible || !this.state.isEmojiPickerVisible) {
+            return;
+        }
+
+        // Dismiss the Keyboard and focus to prevent emoji press twice when selecting an emoji
```

Comment:
Comment could be better,
```
// Dismiss the keyboard to provide a focus for the emoji picker to avoid selection issues.
// Eg: https://github.com/Expensify/App/issues/8110
```
cc: @tgolen

Reply:
Sorry, I don't like this change. If we had a policy of a code comment referencing the GH for each issue that was fixed in the code, then the code will be littered with thousands of them. It's easy to find the associated GH for a change using "blame" to find the PR that introduced the code.

Can you please remove the GH link from the comment?

Reply:
Thanks for your comment, @tgolen. I'm not sure, that's why I put up a `cc:` for you on this.

@mollfpr Sorry for the back and forth, remove the link & let us know.